src/ascend.ts

```diff
@@ -222,7 +222,7 @@ export function ascend(
   pet: Item | undefined = undefined,
   permSkills: Map<Skill, Lifestyle> | undefined = undefined
 ): void {
-  if (!path.classes.includes(playerClass)) {
+  if (playerClass.path !== path) {
```

Comment:
```suggestion
  if (path === Path.none || (path.avatar && playerClass.path !== path))
```

Reply:
I am 80% sure this is what we want

Reply:
oy vey

Reply:
So maybe you want to do
```ts
if (path.avatar && playerClass.path !== path)
```
?

Reply:
That doesn't work. What if I try to ascend journeyman as ed?

I also think we should make sure you can't ascend as Path.none